src/components/field-types/FileFieldComponent.vue

```diff
@@ -1,21 +1,31 @@
 <template>
   <validate :state="fieldState" :custom="{'validate': isValid}">
+    <!-- add hidden input to serve a holder for file input ( input with type file may not contain a value -->
+    <input
+      :id="field.id"
+      :name="field.id"
+      v-model="localValue"
+      v-show="false"
+    />
     <div class="form-group">
-      <label :for="field.id">{{ field.label }}</label>
-    </div>
-
-    <div class="form-group">
+      <label
+        :for="field.id"
+        :class="{ 'is-invalid' : this.fieldState && this.fieldState.$invalid && (this.fieldState.$touched || this.fieldState.$submitted || this.fieldState.$dirty)}"
+      >
+        {{ field.label }}
+      </label>
       <div class="custom-file">
         <label class="custom-file-label" :for="field.id">
           {{ label }}
         </label>
 
         <input
-          :id="field.id"
-          :name="field.id"
+          :id="'alt-' + field.id"
```

Comment:
There may be an attribute with that name, e.g. an entity type with attributes file and alt-file.
I think then the element ids will collide.